api/src/main/java/org/openmrs/module/registrationcore/api/impl/RegistrationCoreServiceImpl.java

```diff
@@ -147,6 +160,23 @@ public Patient registerPatient(Patient patient, List<Relationship> relationships
         return registerPatient(data);
     }
 
+
+	@Override
+	// TODO try to get rid of it
+	public Patient registerPatient(Patient patient, List<Relationship> relationships, String identifierString,
```

Comment:
I'd say we get rid of this now since it has nothing to do with MPI integration

Reply:
Removed